Create appendix of detailed file size estimates - resolves #39 (#54)
fhdsl/NIH_Data_Sharing: Data_size_details.Rmd

# Data File Size Details {-}

> Here you can find more specific information about the file sizes for the types of data commonly generated at Fred Hutch. If you'd like to learn more about the basics of data sizes and computing capacity, please take a look at the Informatics Technology for Cancer Research's class on [Computing for Cancer Informatics](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html)

Reviewer: lol nested acronyms make this difficult :P but I thought it was important to list both the ITCR link and the ITN link.

> Here you can find more specific information about the file sizes for the types of data commonly generated at Fred Hutch. If you'd like to learn more about the basics of data sizes and computing capacity, please take a look at the Informatics Technology for Cancer Research's class on [Computing for Cancer Informatics](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html)

Reviewer: feel free to do this differently if you prefer

> Here you can find more specific information about the file sizes for the types of data commonly generated at Fred Hutch. If you'd like to learn more about the basics of data sizes and computing capacity, please take a look at the Informatics Technology for Cancer Research's class on [Computing for Cancer Informatics](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html)

Author: Works for me!

Whole Exome | Paired Fastq | 3G | highly variable | GEO/SRA | dbGaP/SRA | HS platform dependent |
CRISPR | Single Fastq | ≥500M | highly variable | GEO/SRA | dbGaP/SRA | sgRNA library dependent |
CUT&RUN | Paired Fastq | ≥500M | highly variable | GEO/SRA | dbGaP/SRA | Ab dependent |
CUT&Tag | Paired Fastq | ≥500M | highly variable | GEO/SRA | dbGaP/SRA | Ab dependent |
ChIP-seq | Fastq | 0.5-5G | highly variable | GEO/SRA | dbGaP/SRA | Ab dependent |
ATAC-seq | Fastq | 3-5G | highly variable | GEO/SRA | dbGaP/SRA | |
10x scRNA-seq | Paired Fastq | 10G | highly variable | GEO/SRA | dbGaP/SRA | Target cell number dependent |
10x Multiome | Paired Fastq | ≥20G | highly variable | GEO/SRA | dbGaP/SRA | Target nuclei number dependent |
10x Visium | Paired Fastq | ≥5G | highly variable | GEO/SRA | dbGaP/SRA | Spots under tissue dependent |
Small Genome | Paired Fastq | ≥2G | highly variable | GEO/SRA | N/A | Genome size dependent |
PacBio Amplicon | CCS BAM | 0.5-20G | highly variable | GEO/SRA | N/A | Amplicon size & target depth dependent |
PacBio Small Genome | CCS BAM | highly variable | highly variable | GEO/SRA | N/A | Genome size dependent |

## Imaging Data {-}

> File sizes for medical imaging data vary greatly depending on both the technology used and the organ being imaged. These are some general estimates you can use as a guideline when considering your data management and storage needs. These tables are borrowed from the [Informatics Technology for Cancer Research](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html) training material.

Reviewer: ```suggestion
File sizes for medical imaging data vary greatly depending on both the technology used and the organ being imaged. These are some general estimates you can use as a guideline when considering your data management and storage needs. These tables are borrowed from the [ITN Computing for Cancer Informatics Course](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html).

```

File sizes for medical imaging data vary greatly depending on both the technology used and the organ being imaged. These are some general estimates you can use as a guideline when considering your data management and storage needs. These tables are borrowed from the [ITN Computing for Cancer Informatics Course](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html).

> Here is an table of average file sizes for various medical imaging modalities from @liu_imaging_2017:

Reviewer: did we add this reference to the book.bib too?

Author: No, I am glad you caught that!

ottrpal::include_slide("https://docs.google.com/presentation/d/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0/edit#slide=id.gfb2e21ecdc_0_35")
```
[[source](https://www.mdpi.com/2078-2489/8/4/131)]

Note that depending on the study requirements, several images may be needed for each sample. Thus data storage needs can add up quickly.

```{r, fig.align='center', echo = FALSE, fig.alt= "Example table of overall file storage needs for samples in imaging studies.", out.width="100%"}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0/edit#slide=id.gfb2e21ecdc_0_25")
```

> 

Reviewer: ```suggestion
[[source](https://jhudatascience.org/Computing_for_Cancer_Informatics)]
```

Reviewer: just adding the class as the source here

## Clinical Data {-}

> This information is borrowed from the [Informatics Technology for Cancer Research](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html) training material.

Reviewer: ```suggestion
This information is borrowed from the [ITN Computing for Cancer Informatics Course](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html).

```

This information is borrowed from the [ITN Computing for Cancer Informatics Course](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html).

Really large clinical datasets can also produce sizable file sizes. For example the [Healthcare Cost and Utilization Project (HCUP) National (Nationwide) Inpatient Sample (NIS)](https://www.hcup-us.ahrq.gov/db/nation/nis/nisdbdocumentation.jsp) contains data on more than seven million hospital stays in the United States with regional information.

> According to the NIS website it "enables analyses of rare conditions, uncommon treatments, and special populations" [@NIS].

Reviewer: Maybe we should add the NIS to the book.bib file - is it in there? maybe I missed that, but I noticed there was a question mark showing up in the citation

```{r, fig.align='center', echo = FALSE, fig.alt= "Table of file sizes for the Healthcare Cost and Utilization Project (HCUP) National (Nationwide) Inpatient Sample (NIS) of data from different years and states.", out.width="100%"}
ottrpal::include_slide("https://docs.google.com/presentation/d/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0/edit#slide=id.gfb2e21ecdc_0_42")
```

> ```{r, echo = FALSE}

Reviewer: ```suggestion

```

Reviewer: since we have session info in another part of the appendix we don't really need it here

> devtools::session_info()

Reviewer: ```suggestion

```

> ```

Reviewer: ```suggestion
```
